pkg/adapter/awsperformanceinsightssource/adapter.go

```diff
@@ -102,6 +100,19 @@ func NewAdapter(ctx context.Context, envAcc pkgadapter.EnvConfigAccessor, ceClie
 		mql = append(mql, m)
 	}
 
+	r := rds.New(cfg)
+	var resourceID string
+	dbi, err := r.DescribeDBInstances(&rds.DescribeDBInstancesInput{})
+	if err != nil {
+		logger.Panicf("Unable describe DB clusters: %v", zap.Error(err))
+	}
+
+	for _, instance := range dbi.DBInstances {
+		if *instance.DBInstanceArn == a.String() {
+			resourceID = *instance.DbiResourceId
```

Comment:
Ah ok, I see what you're doing here. Didn't even know you could correlate the 2 IDs like that 👌